x-pack/solutions/security/plugins/security_solution/public/detection_engine/rule_gaps/components/rules_with_gaps_overview_panel/index.tsx

```diff
@@ -123,18 +179,114 @@ export const RulesWithGapsOverviewPanel = () => {
             />
           </EuiPopover>
         </EuiFlexItem>
+
+        {DoubleDividerElement}
+
+        <EuiFlexItem grow={false}>
```

Comment:
do we need show error toast  or warning message instead of component, if request failed?

Currently, when summary requests fails, we would see this:

<img width="1580" alt="Screenshot 2025-05-07 at 11 02 57" src="https://github.com/user-attachments/assets/2955f04a-9b38-4a92-b3ae-328ac5c0293a" />

no error toasts or indications something went wrong, just empty labels an 0% success rate